tests/features/pep695_test.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import annotations
+
+import sys
+
+import pytest
+
+from add_trailing_comma._main import _fix_src
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        pytest.param(
+            'class A[K]:\n'
+            '    def __init__(self, value: K)\n'
+            '        pass\n',
+            id='single line classdef',
+        ),
+        pytest.param(
+            'def notNone[K](v: K) -> K:\n'
```

Comment:
would probably be written as `not_none` rather than camelcase.  also can probably get away with just `: ...` instead of a real function / classdef on all of these